### ISSUE TITLE
Tablist has no focus outline

### ISSUE THREAD

**Issue by xi** [flathub-infra/website#3615]
Keyboard navigation with tabs works great, but the focus is not visible.

Just removing `focus:outline-none` is probably not the best solution though, because then the outline is visible on any `:focus`, not just `:focus-visible`, and also because it looks a but redundant with the underline of the selected item.
